test/javascript/components/common/Editor.test.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { render, fireEvent, waitFor } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import { rest } from 'msw'
+import { setupServer } from 'msw/node'
+import { Editor } from '../../../../app/javascript/components/common/Editor'
+
+const server = setupServer(
+  rest.post('https://exercism.test/submissions', (req, res, ctx) => {
+    return res(
+      ctx.json({ id: 2, tests_status: 'pending', test_runs: [], message: '' })
```

Comment:
@iHiD This is the JSON that I expect from the API. Let me know if we need to change this.

Reply:
What is `message` here? I don't think a submission has a message.

Reply:
Other than that it's good. We could just have a single `test_run` though, rather than `test_runs`. I think we only ever want the JS to expect a single one. The serailizer should probably just choose the latest.

Reply:
I based this from here: https://github.com/exercism/v3-project-management/issues/52#issue-715234576. Is this outdated now?